tests/libTestUtils/TestUtils.cpp

```diff
@@ -150,5 +174,27 @@ FallbackBlockHeader GenerateRandomFallbackBlockHeader() {
                              leaderPubKey, shardId, timestamp, committeeHash);
 }
 
+DS_Comitte_t GenerateRandomDSCommittee(uint32_t size) {
+  DS_Comitte_t ds_c;
+  for (uint32_t i = 1; i <= size; i++) {
+    ds_c.push_front(std::make_pair(GenerateRandomPubKey(), Peer()));
+  }
+  return ds_c;
+}
+
+Shard generateRandomShard(size_t size) {
+  Shard s;
+  for (size_t i = 1; i <= size; i++)
```

Comment:
For consistency, please put for loop statements within curly braces, even if only one statement is contained.

Reply:
Done